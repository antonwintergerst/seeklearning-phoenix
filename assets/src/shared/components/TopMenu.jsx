import React from 'react';

const TopMenu = () => (
  <div>
    <nav className="ui menu">
      <a className="item" href="/#">Home</a>
      <a className="item" href="/#">Courses</a>
      <a className="item" href="/#">Careers</a>
      <a className="item" href="/#">Studying</a>
    </nav>
  </div>
);

export default TopMenu;
