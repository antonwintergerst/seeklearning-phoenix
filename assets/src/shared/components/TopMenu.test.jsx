import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import TopMenu from './TopMenu';

describe('TopMenu', () => {
  it('renders as expected', () => {
    const tree = ReactTestRenderer.create(<div><TopMenu /></div>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
