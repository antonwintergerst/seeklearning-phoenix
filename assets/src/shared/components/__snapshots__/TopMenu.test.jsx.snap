// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopMenu renders as expected 1`] = `
<div
  className="top-menu"
>
  <nav
    className="ui menu container"
  >
    <a
      className="item active"
      href="/#"
    >
      <svg
        height="22px"
        style={
          Object {
            "display": "inline-block",
            "verticalAlign": "middle",
          }
        }
        viewBox="0 0 23 22"
        width="23px"
      >
        <path
          d="M19.462 13.08V22h-5.308v-5.946H8.846V22H4.128v-8.92H0L11.795 0 23 13.08h-3.538"
          style={
            Object {
              "fill": "#fff",
            }
          }
        />
      </svg>
    </a>
    <a
      className="item"
      href="/#"
    >
      Courses
    </a>
    <a
      className="item"
      href="/#"
    >
      Careers
    </a>
    <a
      className="item"
      href="/#"
    >
      Studying
    </a>
  </nav>
</div>
`;
